reference-implementation/test/pipe-to-options.js

```diff
@@ -3,21 +3,34 @@ const test = require('tape-catch');
 // Many other pipeTo-with-options tests have been templated.
 
 test('Piping with no options and a destination error', t => {
```

Comment:
Why did these tests change? (We'll want to summarize any test changes in the commit message since they reflect potential observable behavior changes.)